project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -88,8 +88,38 @@ function c4m_user_notifications_group_status_set_archived($node, $group_former_s
 
 /**
  * Email notification about organic group status change to 'softdelete'.
+ *
+ * @param object $node
+ *   Node of type OG group.
+ * @param string $group_former_status
+ *   Status of OG group, before it was changed to Deleted.
  */
 function c4m_user_notifications_group_status_set_softdeleted($node, $group_former_status) {
+  $admins_ids = c4m_admin_get_admins_ids();
```

Comment:
@soniCaH 
I disagree. The function that is invoked to resolve site admins id is:

`function c4m_admin_get_admins_ids() {
  // Get all site administrators.
  $admin_role = user_role_load_by_name('administrator');
  $results = db_select('users_roles', 'r')
    ->fields('r', array('uid'))
    ->condition('rid', $admin_role->rid, '=')
    ->execute()
    ->fetchAll();

  foreach ($results as &$result) {
    $result = $result->uid;
  }

  return $results;
}`